src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1560,35 +1560,26 @@
      * </pre>
      * <p>parses as:</p>
      * <pre>
-     * +--LITERAL_IF (if)
-     *     |
-     *     +--LPAREN (()
-     *     +--EXPR
-     *         |
-     *         +--IDENT (optimistic)
-     *     +--RPAREN ())
-     *     +--SLIST ({)
-     *         |
-     *         +--EXPR
-     *             |
-     *             +--ASSIGN (=)
-     *                 |
-     *                 +--IDENT (message)
-     *                 +--STRING_LITERAL ("half full")
-     *         +--SEMI (;)
-     *         +--RCURLY (})
-     *     +--LITERAL_ELSE (else)
-     *         |
-     *         +--SLIST ({)
-     *             |
-     *             +--EXPR
-     *                 |
-     *                 +--ASSIGN (=)
-     *                     |
-     *                     +--IDENT (message)
-     *                     +--STRING_LITERAL ("half empty")
-     *             +--SEMI (;)
-     *             +--RCURLY (})
+     * LITERAL_IF -&gt; if
```

Comment:
I think it should be `--LITERAL_IF` and next lines should have additional indents

I took as en example [TokenTypes#METHOD_DEF](https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java#L198)

Reply:
I have worked on similar issues and I was always asked to remove additional indents by @pbludov , I can additional indents if you want.

Reply:
We have come to an agreement to keep `--`.  So it should start with `--LITERAL_IF`

Reply:
Let's remove "--".
All already merged cases can be fixed in separate issue.

Reply:
I have removed "--" and all additional indents.